### DISCUSSION TITLE
2023.02.06 Mw7.8 Turkey earthquake LOS deformation

### DISCUSSION THREAD

**Discussion by BelledDeng** [gmtsar/gmtsar#796] [Q&A]
Dear Xiaohua-Eric-Xu,
I used ascending(2023/01/28-2023/02/09) and descending(2023/02/09) data to get the 2023.02.06 Mw7.8 Turkey earthquake coseismic LOS deformation. The range of deformation is about -0.5~2m. 
![截图20230901160415](https://github.com/gmtsar/gmtsar/assets/105110883/370f20db-4ea9-471d-833c-6956c7fbaee0)

However, the reported LOS deformation of this earthquake is about -4~4m,like these:
![截图20230901155823](https://github.com/gmtsar/gmtsar/assets/105110883/fd762189-f732-4926-8a76-c4d197dc7445)
An, Q.; Feng, G.; He, L.;Xiong, Z.; Lu, H.; Wang, X.; Wei, J. Three-Dimensional Deformation of the 2023 Turkey Mw 7.8 and Mw 7.7 Earthquake Sequence Obtained by Fusing Optical and SAR Images. Remote Sens. 2023, 15, 2656. https://doi.org/10.3390/rs15102656
![截图20230901160208](https://github.com/gmtsar/gmtsar/assets/105110883/bd734d42-b894-4f8a-b73d-6b41029e92c7)
Mai, P. M., T. Aspiotis, T. A. Aquib, E. V. Cano, D. Castro-Cruz, A. Espindola-Carmona, B. Li, X. Li, J. Liu, R. Matrau, et al. (2023). The Destructive Earthquake Doublet of 6 February 2023 in South-Central Türkiye and Northwestern Syria: Initial Observations and Analyses, The Seismic Record. 3(2), 105–115, doi: 10.1785/0320230007.

I don't know how to deal with it.Could you please give me some suggestions?

**Comment by Xiaohua-Eric-Xu** [comment]
make_a_offset.csh will produce a freq_xcorr.dat that contains both range and azimuth offset estimates. Please use the attached figure to make a plot. You may want to use the nx and ny values suggested in the script for running make_a_offset.csh.

[make_xcorr_plot.csh.txt](https://github.com/gmtsar/gmtsar/files/12535113/make_xcorr_plot.csh.txt)

**Reply by Elcappo2019** [reply]
Dear Xiaohua-Eric-Xu,

1- In the script "make_a_offset.csh", which generates the "freq_xcorr.dat" file containing range and azimuth offset estimates, there are parameters denoted as **nx and ny.** 
**How can these parameters be appropriately modified within the script to meet specific requirements?**
2-Executing make_xcorr_plot.csh Script on freq_xcorr.dat:
Following the generation of the **freq_xcorr.dat** file, I intend to utilize the script named **make_xcorr_plot.csh** (or potentially make_xcorr_plot.csh.txt). **Should this script be executed in the same directory as **freq_xcorr.dat,** or is there a preferred alternative location?**

**Reply by Xiaohua-Eric-Xu** [reply]
1. I think this may require some knowledge on the satellite data you are processing. You'll need to know how many range and azimuth samples there are and what are the pixel spacing. Then you can choose proper size for sort of a square search window and sampling numbers along each direction.
2. You could plot maps by running the script at the location where freq_xcorr.dat exist.
